src/js/actions/export.js

```diff
@@ -374,9 +374,15 @@ define(function (require, exports) {
      * Adds an asset, or assets to the end of the document asset list, or that of a layer or layers
      * If props not provided, choose the next reasonable scale and create an otherwise vanilla asset
      *
+     * Recognizes an empty list of layers as implying doc-level export
```

Comment:
Not a huge fan of using an empty list to denote doc-level exports, but maybe it makes sense in the bigger picture.